libraries/chain/controller.cpp

```diff
@@ -2274,7 +2282,10 @@ struct controller_impl {
 
       db.modify(gpo, [&proposed_participants, cur_block_num](auto& gp) {
          gp.proposed_security_group_block_num    = cur_block_num;
-         gp.proposed_security_group_participants = std::move(proposed_participants);
+         gp.proposed_security_group_participants = {proposed_participants.begin(),
+                                                    proposed_participants.end(),
+                                                    gp.proposed_security_group_participants.key_comp(),
+                                                    gp.proposed_security_group_participants.get_allocator()};
```

Comment:
same here regarding explicitly specifying allocator and comparator

Reply:
The global_property_object ctor macro sets the allocator, so the comparator has to come along for the ride since it cannot be defaulted if the allocator is not defaulted

Reply:
thanks, it was not straightforward, now I see it. it is `chainbase::allocator<Allocator>`